### PR TITLE
8 GET reviews added

### PR DESCRIPTION
**Pull request by panicroll**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -247,5 +238,56 @@ describe("GET /api/reviews/:review_id with comment_count", () => {
...
+          expect(review).toEqual(
+            expect.objectContaining({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              review_body: expect.any(String),
```
Comment (two-triangles) [inline __tests__/app.test.js]: We aren't going to expect the revie_body back

File: models/fetchReviews.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const fetchReviews = () => {
+  return db
+    .query(
+      `SELECT reviews.*, COUNT(comments.comment_id)::INT AS comment_count 
+    FROM reviews LEFT JOIN comments ON comments.review_id = reviews.review_id 
+    GROUP BY reviews.review_id ORDER BY created_at desc;`
+    )
+    .then(({ rows }) => {
+        console.log(rows)
```
Comment (two-triangles) [inline models/fetchReviews.js]: remove console.logs that you do not need

**Comment by two-triangles** [comment]
a few things to change but looking good for the most part

File: __tests__/app.test.js (hunk 1 above)
Comment (panicroll) [reply __tests__/app.test.js]: It was mentioned on the ticket so I thought it needed to be added there, I'll remove it!

**Review by CatHoang — APPROVED** [review]
looks good, feel free to merge

**Pull request merged**